salt/pillar/file_tree.py

```diff
@@ -158,6 +158,7 @@
 import salt.utils.stringio
 import salt.template
 from salt.ext import six
+from salt.utils.stringutils import to_str
```

Comment:
please don't import functions like this, it will add it to the functions in the modules.

```
import salt.utils.stringutils
```